tensorflow_lite_support/python/test/task/vision/image_classifier_test.py

```diff
@@ -126,7 +168,138 @@ def test_classify_model(self, model_file_type, max_results,
+
+    categories = image_result_dict['classifications'][0]['classes']
+
+    self.assertLessEqual(
+      len(categories), _MAX_RESULTS, 'Too many results returned.')
+
+  def test_score_threshold_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, score_threshold=_SCORE_THRESHOLD)
+
+    # Loads image.
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    categories = image_result_dict['classifications'][0]['classes']
+
+    for category in categories:
+      score = category['score']
+      self.assertGreaterEqual(
+        score, _SCORE_THRESHOLD,
+        'Classification with score lower than threshold found. {0}'.format(
+          category))
+
+  def test_allowlist_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, class_name_allowlist=_ALLOW_LIST)
+
+    # Loads image.
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    categories = image_result_dict['classifications'][0]['classes']
+
+    for category in categories:
+      label = category['className']
+      self.assertIn(
+        label, _ALLOW_LIST,
+        'Label "{0}" found but not in label allow list'.format(label))
+
+  def test_denylist_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, score_threshold=0.01, class_name_denylist=_DENY_LIST)
+
+    # Loads image
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    categories = image_result_dict['classifications'][0]['classes']
+
+    for category in categories:
+      label = category['className']
+      self.assertNotIn(label, _DENY_LIST,
+                       'Label "{0}" found but in deny list.'.format(label))
+
+  def test_combined_allowlist_and_denylist(self):
+    # Fails with combined allowlist and denylist
+    with self.assertRaisesRegex(
+        Exception,
+        r"INVALID_ARGUMENT: `class_name_whitelist` and `class_name_blacklist` "
+        r"are mutually exclusive options. "
+        r"\[tflite::support::TfLiteSupportStatus='2'\]"):
+      base_options = _BaseOptions(model_file=_ExternalFile(file_name=self.model_path))
```

Comment:
Please follow the google c++ style guide https://google.github.io/styleguide/cppguide.html#Line_Length: Each line of text in your code should be at most 80 characters long.

Please keep in mind to follow the google c++ style guide. Thanks! :)

Reply:
@ziyeqinghan Thanks for the reference to the guide and the line-length limit used by Google! I'll hard wrap them to 80 from now on.